sdk/entra/ci.yml

```diff
@@ -0,0 +1,43 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+
+trigger:
+  branches:
+    include:
+    - main
+    - hotfix/*
+    - release/*
+  paths:
+    include:
+    - sdk/entra/WebJobs.Extensions.AuthenticationEvents/
+#if (false)
+    # The following paths should only be included in template/ci.yml, and removed from any other SDKs which copy this file.
+    # The surrounding conditions should accomplish that when installed with `dotnet new azsdk`.
+    # eng/common code changes trigger template pipeline for basic checking.
+    - eng/common/
+#endif
```

Comment:
Please remove these lines. Normally the new project template should handle this.

```suggestion
```